src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCFUnitTest.java

```diff
@@ -110,6 +110,12 @@ public void testLowQualVariantToGQ0HomRef() {
         Assert.assertTrue(!modified.filtersWereApplied());
         Assert.assertEquals(modified.getLog10PError(), VariantContext.NO_LOG10_PERROR);
 
+        final Genotype longPls = VariantContextTestUtils.makeG("sample1", Allele.NO_CALL, Allele.NO_CALL, 0,0,0,0,0,0);
+        final VariantContext lotsOfZeroPls = makeNoDepthVC("lowQualVar", Arrays.asList(LONG_REF, DELETION, Allele.NON_REF_ALLELE), LONG_REF.length(), longPls);
+        final VariantContext properlySubset = reblocker.lowQualVariantToGQ0HomRef(lotsOfZeroPls).make();
+        Assert.assertEquals(properlySubset.getGenotype(0).getPL().length, 3);
```

Comment:
Can you check the content of PL also is correnct? is it all 0s or '.'?

Reply:
Assert.assertEquals(...getPL(), new int [] { 0, 0, 0}) may just work.

Reply:
It does work, thanks.  This is done.